.github/workflows/publish-release.yml

```diff
@@ -115,16 +119,20 @@ jobs:
       - uses: actions/checkout@v3
       - name: install
         run: npm ci
+      - name: get version
+        run: |
+          export PACKAGE_VERSION=$(node -e "(function () { console.log(require('./package.json').version) })()") \
+          export CDN_VERSION=$(cat VERSION)
```

Comment:
Does exporting the variable work in a later step. What I have seen, we need to use an output variable.

https://stackoverflow.com/a/75659156

Reply:
I figured it should work if its in the same job, since it doesn't spin up a new process, but we should verify

Reply:
moved to use @actions/core and reference in steps outputs